framework/doc/content/syntax/MFEM/index.md

```diff
@@ -0,0 +1,83 @@
+
+If the integrator you wish to use is available, you can specify it in the `type` parameter simply by taking its MFEM integrator name, swapping the word `Integrator` for `Kernel`, and prepending MFEM to the beginning of its name. The table below shows a few examples of this naming convention:
+
+| MFEM name      | MFEM-MOOSE name      |
+| ------------- | ------------- |
+| `DiffusionIntegrator` | `MFEMDiffusionKernel` |
+| `CurlCurlIntegrator` | `MFEMCurlCurlKernel` |
+| `VectorDomainLFIntegrator` | `MFEMVectorDomainLFKernel` |
+
+Putting this together, our `Kernels` block might look as follows:
+
+!listing test/tests/mfem/kernels/diffusion.i block=/Kernels
+
+Now we set up boundary conditions. The full list of boundary conditions available may be found in the [BCs directory](source/mfem/bcs). Here, we choose scalar Dirichlet boundary conditions, which correspond to the [MFEMScalarDirichletBC.md] type.
+
+!listing test/tests/mfem/kernels/diffusion.i block=/BCs
+
+### Solver and Executioner
+
+With the equation system set up, we specify how it is to be solved. Firstly, we choose a preconditioner and solver. The list of available types may be found in the [solvers directory](source/mfem/solvers). For problems with high polynomial order, setting [!param](/Solver/MFEMHypreGMRES/low_order_refined) to `true` may greatly increase performance, as explained [here](MFEMSolverBase.md).
+
+While in principle any solver may be used as main solver or preconditioner, the main limitation to keep in mind is that Hypre solvers may only be preconditioned by other Hypre solvers. Furthermore, when a Hypre solver has its `low_order_refined` parameter set to `true`, it ceases to be considered a Hypre solver for preconditioning purposes.
+
+!listing test/tests/mfem/kernels/diffusion.i block=/Preconditioner Solver remove=jacobi
+
+Static and time-dependent executioners may be implemented respectively with the [MFEMSteady.md] and [MFEMTransient.md] types. If MFEM-MOOSE has been built with GPU offloading capabilities, it is possible to set [!param](/Executioner/MFEMSteady/device) to `cuda` or `hip` to make use of GPU acceleration. For GPU runs, it is advisable to choose an [!param](/Executioner/MFEMSteady/assembly_level) other than `legacy`, otherwise the matrix assembly step will not be offloaded. The options for [!param](/Executioner/MFEMSteady/assembly_level) are `legacy`, `full`, `element`, `partial`, and `none` (the latter is only available if MFEM-MOOSE has been built with libCEED support).
```

Comment:
What does `none` mean? I thought that the main target of libCEED was also partial assembly

Reply:
"NONE, "Matrix-free" form that computes all of its action on-the-fly without any substantial storage. In this case [the matrix of quadrature values] D is computed on the fly, this format is also significantly faster than the matrix formats, but is currently slower than partial assembly due to the increased number of computations. However, in the case of operators that need to be reassembled frequently this assembly level might be faster than partial assembly by skipping any reassembly steps."